target/linux/tegra/base-files/lib/tegra.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
Yes there is valid model in /tmp/sysinfo/model, but during creation of image the metadata: board is set to tegra. When I did sysupgrade it failed the metadata check (it is not mandatory, so it did proceed further). That is why this script is sourced only on sysupgrade.